src/custom/components/OrderProgressBar/index.tsx

```diff
@@ -363,84 +359,3 @@ function useGetProgressBarInfo({
     isPending: orderIsPending,
   }
 }
-
-// TODO: Make a dumb component and Cosmos preview for different
-const ExecutedWrapper = styled.div`
-  display: flex;
-  align-items: center;
-  padding-bottom: 1rem;
-
-  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
-    font-size: 0.8rem;
-  `};
-
-  img {
-    padding: 1rem;
-    margin-right: 10px;
-
-    ${({ theme }) => theme.mediaWidth.upToExtraSmall`
-      padding: 0;
-      max-width: 60px;
-    `};
-  }
-
-  a {
-    margin: 0;
-    margin-top: 15px;
-    display: block;
-  }
-
-  > div > div {
-    margin-bottom: 5px;
-  }
-`
-
-const Strong = styled.strong`
-  font-size: 0.9rem;
-  white-space: nowrap;
-`
-
-function TransactionExecutedContent({
```

Comment:
Moved this to separate component `src/custom/components/TransactionExecutedContent/index.tsx`